src/Tizen.NUI/src/public/BaseComponents/GLView.cs

```diff
@@ -55,9 +55,20 @@ internal GLView(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cMemor
         /// <summary>
         /// Creates an initialized GLView.
         /// </summary>
+        /// <param name="">The format of the color buffer</param>
+        /// <since_tizen> 10 </since_tizen>
+        public GLView(ColorFormat colorFormat) : this(Interop.GLView.New((int)colorFormat, (int)GLBackendMode.DEFAULT), true)
+        {
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+        
+        /// <summary>
+        /// Creates an initialized GLView.
+        /// </summary>
+        /// <param name="backendMode">The backend mode to be used with GLView (Direct Rendering or EGL Image)</param>        
         /// <param name="colorFormat">The format of the color buffer</param>
         /// <since_tizen> 10 </since_tizen>
```

Comment:
remove this since tizen, as
public GLView(ColorFormat colorFormat) : this(Interop.GLView.New((int)colorFormat, (int)GLBackendMode.DEFAULT), true)
is already up there and this constructor newly added.

Reply:
We are planning to be public the DirectRenderingMode on the next Tizen version.
So you don't need to ACR now. Just tag [EditorBrowsable(EditorBrowsableState.Never)] please.
This was confirmed by @hyunjushin.